### DISCUSSION TITLE
Welcome to homebridge-hubitat-tonesto7 Discussions!

### DISCUSSION THREAD

**Discussion by tonesto7** [tonesto7/homebridge-hubitat-tonesto7#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MattLMiller86** [comment]
Is there a way to show my Hombridge  devices in to Hubitat?

**Reply by spikenheimer** [reply]
this might be better asked on the Hubitat community boards - https://community.hubitat.com - and probably has already been asked/answered in some fashion

**Reply by tonesto7** [reply]
Anything is possible but it would require more work than you can imagine.  There is a high likelihood that the devices in Homebridge also have an integration in Hubitat.

**Comment by finepl** [comment]
I am struggling to get a QR code in the Homebridge V2 app for my Hubitat C7.  You mention installing [homebridge-config-ui-x].  I am baffled by this process.  How do you do this on the Hubitat? Is this for Mac OS, not Hubitat? I’m completely lost and need step-by-step instructions for this part of the process, please!

Thanks for your help.
